app/services/push_notification/reproof_completed_event.rb

```diff
@@ -0,0 +1,30 @@
+module PushNotification
+  class ReproofCompletedEvent
+    EVENT_TYPE = 'https://schemas.openid.net/secevent/risc/event-type/reproof-completed'.freeze
```

Comment:
reproof is a custom login.gov event, so can we change the domain like we do for the other custom events?

https://schemas.login.gov/secevent/risc/event-type/authorization-fraud-detected


ex:
```
https://schemas.login.gov/secevent/risc/event-type/authorization-fraud-detected
```